x-pack/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -52,11 +53,31 @@ interface Props {
 
 export class Distribution extends Component<Props> {
   public formatYShort = (t: number) => {
-    return `${t} ${unitShort(this.props.urlParams.transactionType)}`;
+    return i18n.translate(
+      'xpack.apm.transactionDetails.transactionsDurationDistributionChart.unitShortLabel',
+      {
+        defaultMessage:
+          '{transCount} {transType, select, request {req.} other {trans.}}',
```

Comment:
I don't understand this line ` {transType, select, request {req.} other {trans.}}`. 
Does it mean that if `transType` is "request" then use "req." else use "trans."?

Reply:
Yes. It's [ICU Message syntax](http://userguide.icu-project.org/formatparse/messages). `select` statement is a kind of switch. 
You can use this https://format-message.github.io/icu-message-format-for-translators/editor.html to test various ICU messages (including `select` statements)